### PR TITLE
Update nextcloud-vue to version 4.0.0

### PR DESCRIPTION
**Pull request by marcoambrosini**
Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

**Comment by PVince81** [comment]
some Jest error, do we need to fix some imports ? or is it related to node v12 ?

File: webpack.common.js [hunk 1]
```diff
@@ -108,6 +109,8 @@ module.exports = {
 		new StyleLintPlugin({
 			files: ['**/*.vue'],
 		}),
+		// Make appName & appVersion available as a constant
+		new webpack.DefinePlugin({ appName: JSON.stringify('talk') }),
```
Comment (PVince81) [inline webpack.common.js:113]: where is appVersion ?

also, for the tests you might need to add values in "src/test-setup.js"

**Comment by PVince81** [comment]
I'll have a look at the test issue, seems it's another lib that need to be transpiled

**Comment by PVince81** [comment]
@ma12-co okay, tests are fixed, but see my comment above.

Also strange, the event-bus warning still appears for the tests:
```
Proxying an event bus of version 1.3.0 with 2.0.0
```
I thought this one would disappear with the 4.0.0 update but probably will only once all the other libs get the 4.0.0 update, too.
Never mind that message, then.

**Comment by PVince81** [comment]
okay, tests are green now

File: webpack.common.js (hunk 1 above)
Comment (ma12-co) [reply webpack.common.js:113]: oops

**Comment by ma12-co** [comment]
thanks @PVince81

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by danxuliu** [comment]
This breaks the chat view for me. A scroll bar is no longer shown, so when there are more messages than the page height the new message field is pushed down and not visible.

Do I need to use a specific node/npm version to build now? Or is it a general issue?

**Comment by nickvergessen** [comment]
Will be fixed with 4.0.1 IIRC

**Pull request merged**